test-framework/core/src/main/java/org/keycloak/test/framework/annotations/InjectKeycloakUrls.java

```diff
@@ -0,0 +1,12 @@
+package org.keycloak.test.framework.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface InjectKeycloakUrls {
```

Comment:
Nice -> I was actually thinking about it, on how to enable the option to get the server url.